webextensions/api/webRequest.json

```
@@ -63,7 +63,8 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/webRequest/CertificateInfo",
             "support": {
               "chrome": {
-                "version_added": false
+                "version_added": false,
+                "notes": "There was <a href='https://crrev.com/c/644858'>an attempt</a> to implement this."
```

I don't think this text works in the context of BCD. It's certainly true, but I don't think it's especially relevant to web developers who want to use this feature. Instead, we could link to [the corresponding bug](https://bugs.chromium.org/p/chromium/issues/detail?id=628819), so that if there's ever progress implementing this feature they'd be made aware of it more directly. We'd typically present it like this:

```suggestion
                "notes": "See <a href='https://crbug.com/628819'>bug 628819</a>."
```

---

I noticed that the bug also points to the Code Review link I used, so this would not lose any information. Thank you for finding the bug!